tests/hook_main.py

```diff
@@ -15,6 +15,7 @@
 # imports from factories
 from app.factories.user import UserFactory
 from app.factories.notification import NotificationFactory
+from app.factories.event import EventFactoryBasic
```

Comment:
This warning is to be ignored in the hook files always. Imports cannot be performed without setting sys.path a few lines above.